sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/GeneratedMutableEvaluationSuite.scala

```diff
@@ -44,14 +43,6 @@ class GeneratedMutableEvaluationSuite extends ExpressionEvaluationSuite {
 
     val actual = plan(inputRow)
     val expectedRow = new GenericRow(Array[Any](expected))
-    if (actual.hashCode() != expectedRow.hashCode()) {
```

Comment:
The hash of UTF8String is different than String, this will fail, or will have to use UTF8String in expectedRow (updating all the tests). Which one do you like?